src/API/Run.hs

```diff
@@ -41,9 +42,11 @@ _handleParsed (SpielCommand _ gameFile replExpr buf _) game =
          H (Left er)    -> [er]
          H (Right buf') -> [progTypes, (serverRepl game gameFile replExpr (buf', [], 1))]
    else
-      progTypes : map (SpielTypeError . snd) (errors progTCRes)
```

Comment:
Reverses the order of type errors so that the first type error in the program is presented to the user rather than the last one, which is often `you did not define x`